src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Movies = {
```

Comment:
it's better to name it `Props`
means properties of the component
```suggestion
type Props = {
```